pkgs/tools/misc/kb/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "kb";
+  version = "0.1.7";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "gnebbia";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0hjvf1ycwf4f8rnrshj30ij56mkji2ic3ibpschyvng5jslh1h9b";
+  };
+
+  preBuild = ''
+    substituteInPlace setup.py \
+      --replace \
+        "install_requires=[\"colored\",\"toml\",\"attr\",\"attrs\",\"gitpython\"]," \
+        "install_requires=[\"colored\",\"toml\",\"attrs\",\"gitpython\"],"
+  '';
+
+  propagatedBuildInputs = with python3Packages; [
+    colored
+    toml
+    attrs
+    gitpython
+  ];
+
+  nativeCheckInputs = [ python3Packages.pytest ];
+
+  doCheck = true;
+
```

Comment:
I don't think that does anything for python packages

```suggestion
```

Reply:
Yep, `doCheck` defaults to true inside `buildPythonPackage`.